CMakeLists.txt

```diff
@@ -29,8 +29,11 @@ option(ENABLE_OPROFILE "enable oprofile support" OFF)
 option(ENABLE_SELF_HOST "use pyston to test pyston" OFF)
 option(ENABLE_VALGRIND "pyston valgrind support" OFF)
 
+option(ENABLE_PGO "enable -fprofile-generate/-fprofile-use" OFF)
+option(ENABLE_LTO "enable -flto (only if ENABLE_PGO=ON as well)" OFF)
+
 # automatically use ccache if found
-if(ENABLE_CCACHE)
+if(ENABLE_CCACHE AND NOT ENABLE_PGO)
```

Comment:
I did see .gcda files in my ~/.cache directory, which is when I disabled it.  Those might have come from previous builds, though (before I added the PROFILE_DIR support).  I can try and disable it only for `use` and see if they appear again.